app/components/ui/molecules/DashboardListItem.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { Flex, Box } from 'grid-styled'
+import { differenceInCalendarDays, format } from 'date-fns'
+import { th } from '@pubsweet/ui-toolkit'
+import PropTypes from 'prop-types'
+
+const dashboardStatusColor = (status, theme) =>
+  ({
+    1: theme.colorTextSecondary,
+    2: theme.colorError,
+    3: theme.colorSuccess,
+  }[status])
+
+const dashboardStatusText = status =>
+  ({
+    1: 'Waiting for decision',
+    2: 'Rejected',
+    3: 'Approved',
+  }[status])
+
+const dashboardDateText = date => {
+  const diffDays = differenceInCalendarDays(new Date(), date)
+  if (diffDays < 0) {
+    return 'Invalid date'
+  }
+  if (diffDays === 0) {
+    return 'Today'
+  }
+  if (diffDays === 1) {
+    return 'Yesterday'
+  }
+  return `${diffDays} days ago`
```

Comment:
I believe this is meant to switch to "1 month ago" etc. at some point. Not sure where I saw that.

Reply:
This is what I remember as well. It would be nice if we find that invision page to link it in the github issue. Was is just months and then years ago?

Reply:
Ideally it would be one of:
"Today"
"Yesterday"
"x days ago"
"x months ago" 

I know this could get complex calculating no. of days per month. I think rounding down to nearest multiple of 30 days would suffice for now for "x months ago", unless there's a library or similar that can do it more accurately without lots of work. 

We have the actual date below so it doesn't have to be 100% accurate.